aws/resource_aws_lambda_event_source_mapping.go

```diff
@@ -35,13 +35,12 @@ func resourceAwsLambdaEventSourceMapping() *schema.Resource {
 			},
 			"starting_position": {
 				Type:     schema.TypeString,
-				Required: true,
+				Optional: true,
 				ForceNew: true,
 			},
 			"batch_size": {
 				Type:     schema.TypeInt,
 				Optional: true,
-				Default:  100,
```

Comment:
As mentioned, this change will likely require adding a `DiffSuppressFunc` to handle which API value is being returned depending on the event source type.

Reply:
Done.  Thanks!

Reply:
Makes sense.  Just to check, but you're going to make that change, right?  Or would you prefer i do it?